src/sentry/issues/endpoints/organization_group_search_view_starred_order.py

```diff
@@ -61,33 +61,14 @@ def put(self, request: Request, organization: Organization) -> Response:
 
         try:
             with transaction.atomic(using=router.db_for_write(GroupSearchViewStarred)):
```

Comment:
Hmm I wonder if this transaction is necessary anymore since we are just doing a single bulk update